code/modules/events/rogue_drones.dm

```diff
@@ -38,14 +38,16 @@
 /datum/event/rogue_drone/end()
 	var/num_recovered = 0
 	for(var/mob/living/simple_animal/hostile/retaliate/malf_drone/D in drones_list)
```

Comment:
It's despawning them by calling qdel() yeah, I changed it from the broken code that existed before that I think made them not get qdel'd properly.